src/Markdig/Renderers/TextRendererBase.cs

```diff
@@ -128,6 +128,11 @@ protected internal void Reset()
                 ThrowHelper.InvalidOperationException("Cannot reset this TextWriter instance");
             }
 
+            ResetInternal();
+        }
+
+        internal void ResetInternal()
```

Comment:
There is now a split between an exposed `Reset` and internal `ResetInternal`.

```c#
protected internal void Reset()
{
    // Reset Writer

    ResetInternal();
}

internal void ResetInternal()
{
    // Reset internal state
}
```

We only use `ResetInternal` in the library as we can't reset an arbitrary writer.

Would it make sense to change the exposed `Reset` to not reset the writer and leave that to the caller instead?

Reply:
Don't have enough insights on the implication, so I trust your choice 😉